pkg/logs/decoder/line_wrapper.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package decoder
+
+import (
+	"bytes"
+)
+
+// LineUnwrapper removes all the extra information that were added to the original log
+type LineUnwrapper interface {
+	Unwrap(line []byte) []byte
+}
+
+// IdentityUnwrapper does nothing
+type identityUnwrapper struct{}
+
+// Unwrap returns line
+func (u identityUnwrapper) Unwrap(line []byte) []byte {
+	return line
+}
+
+// NewUnwrapper returns a default LineUnwrapper that does nothing
+func NewUnwrapper() LineUnwrapper {
+	return &identityUnwrapper{}
+}
+
+// headerLen represents the length of the header of docker logs
+// see here for more information: https://godoc.org/github.com/moby/moby/client#Client.ContainerLogs
+const headerLen = 8
+
+// DockerUnwrapper removes all the information added by docker to logs coming from containers
+type DockerUnwrapper struct{}
+
+// NewDockerUnwrapper returns a new DockerUnwrapper
+func NewDockerUnwrapper() *DockerUnwrapper {
+	return &DockerUnwrapper{}
+}
+
+// Unwrap removes the header and the timestamp from container logs
+func (u DockerUnwrapper) Unwrap(line []byte) []byte {
+	if len(line) < headerLen {
```

Comment:
why not reusing `docker.parseMessage` here? Code looks good but I'm worried of code duplication, and we'd update one method and not the other

Reply:
To avoid circular import which would cause the code not to compile. We could create a `(docker|utils|common package)` at  the root of `pkg/logs` instead.

Reply:
yeah that'd be preferable

Reply:
done